geometry_msgs/PoseStamped target_pose
float32 speed
float32 distance
bool exploration
bool reverse_allowed
